# Message describing the full Topological Navigation Map

TopoNavNodeMsg[] nodes
TopoNavEdgeMsg[] edges
